02.wallets/article.md

```diff
@@ -67,6 +67,19 @@ The [Arctic wallet](https://arcticwallet.io/) is a multi asset wallet that works
 * Non-Custodial
 * In-app exchange
 
+**Cons:**  
+* Staking not available
+* Masternodes not available
+* Shielded transactions not available
+[/tab]
+[tab title="Coinomi"]
+The [Coinomi](https://www.coinomi.com/) is a multi asset wallet that works on desktop or mobile hardware.  
```

Comment:
It currently reads "The Coinomi is a multi asset wallet [...]", perhaps it should be:
```suggestion
The [Coinomi](https://www.coinomi.com/) wallet is a multi asset wallet that works on desktop or mobile hardware.  
```

---

**Additionally... in my opinion:** due to quite a lot of shady history and **[extremely bad security practices](https://www.avoid-coinomi.com/)**, there should perhaps be a warning about Coinomi regarding safety and security.